src/pages/VenuePage/VenuePage.tsx

```diff
@@ -187,13 +195,16 @@ const VenuePage: React.FC = () => {
     return <>This venue does not exist</>;
   }
 
-  if (!venue || !venueId) {
+  if (!venue || !venueId || !profile) {
     return <LoadingPage />;
   }
```

Comment:
I think `profile` may have been explicitly removed from here recently, potentially as it was causing a bug or similar?

@mike-lvov might knowe/remember more

Reply:
I don't know why this was removed but right now it's problematic and the login is broken because of this. When you login `user` exists but `profile` is undefined and the `VenuePage.tsx` redirects you to the entrance page.

Reply:
Seems my memory fails me, looking at the blame on `staging`, this seemingly hasn't been touched for a few months: 

- https://github.com/sparkletown/sparkle/blame/staging/src/pages/VenuePage/VenuePage.tsx#L183

![image](https://user-images.githubusercontent.com/753891/107012746-89199f80-67ed-11eb-98d3-9db208a5b71b.png)